# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1357134677 -3600
#      Wed Jan 02 14:51:17 2013 +0100
# Node ID 596b26e11ddb68e82aede4ae407e4039196895ff
# Parent  84b38d6442e685ebc1f57f309f3c43f0c8c55447
fix bug #38010 struct2cell documentation error

diff --git a/libinterp/octave-value/ov-cell.cc b/libinterp/octave-value/ov-cell.cc
--- a/libinterp/octave-value/ov-cell.cc
+++ b/libinterp/octave-value/ov-cell.cc
@@ -1394,17 +1394,17 @@ If @var{f} is the number of fields in th
 cell array will have a dimension vector corresponding to\n\
 @code{[@var{F} size(@var{S})]}.  For example:\n\
 \n\
 @example\n\
 @group\n\
 s = struct (\"name\", @{\"Peter\", \"Hannah\", \"Robert\"@},\n\
            \"age\", @{23, 16, 3@});\n\
 c = struct2cell (s)\n\
-   @result{} c = @{1x1x3 Cell Array@}\n\
+   @result{} c = @{2x1x3 Cell Array@}\n\
 c(1,1,:)(:)\n\
    @result{}\n\
       @{\n\
         [1,1] = Peter\n\
         [2,1] = Hannah\n\
         [3,1] = Robert\n\
       @}\n\
 c(2,1,:)(:)\n\
